snippets/Google.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -12,33 +12,212 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+using Google.Apis.Download;
+using Google.Apis.Storage.v1;
+using Google.Apis.Storage.v1.Data;
+using Google.Apis.Upload;
+using System;
+using System.IO;
 using System.Linq;
+using System.Text;
 using Xunit;
 
 namespace Google.Storage.V1.Snippets
 {
     [Collection(nameof(StorageSnippetFixture))]
     public class StorageClientSnippets
     {
-        private readonly StorageSnippetFixture _fixture;
+        private readonly StorageSnippetFixture fixture;
 
         public StorageClientSnippets(StorageSnippetFixture fixture)
         {
-            this._fixture = fixture;
+            this.fixture = fixture;
         }
 
         [Fact]
-        public void ListObjects()
+        void ListBuckets()
```

Comment:
To which code formatter are you referring? Is it part of the repo or an OSS tool w/ config specified for contributors that I've missed?